README.md

```diff
@@ -1,17 +1,269 @@
 # avm_scene
 
-avm_scene adds graphical capabilities on top of `gen_server`, such as pushing graphical updates on
-`handle_info` or `handle_call`.
+A GenServer wrapper that adds graphical display capabilities to AtomVM applications.
+
+## Overview
+
+`avm_scene` extends the standard `gen_server` behavior by automatically handling display updates
+when callbacks return `{:push, display_list}` directives. It acts as a bridge between your
```

Comment:
Should this be `{:push, display_list}` >> `[{:push, display_list}]`?

Reply:
A directive is a tuple. Directives is a list of directive. However it was confusing, changed.